tests/Integration/Service.Tests/StockTriggerLevelsModuleSpecs/WhenAdding.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Linn.Stores.Service.Tests.StockTriggerLevelsModuleSpecs
+{
+    using System.Collections.Generic;
+
+    using FluentAssertions;
+    using Linn.Common.Facade;
+    using Linn.Stores.Domain.LinnApps.StockLocators;
+    using Linn.Stores.Resources;
+
+    using Nancy;
+    using Nancy.Testing;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenAdding : ContextBase
+    {
+        private StockTriggerLevel requestResource;
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.requestResource = new StockTriggerLevel
+                                        {
+                                            PartNumber = "PART",
+                                            Id = 1,
+                                            KanbanSize = 1,
+                                            LocationId = 1,
+                                            MaxCapacity = 1,
+                                            PalletNumber = 1,
+                                            TriggerLevel = 1
+                                        };
+            var stockTriggerLevel = new StockTriggerLevel
+                                        { 
+                                            PartNumber = "PART",
+                                            Id = 1,
+                                            KanbanSize = 1,
+                                            LocationId = 1,
+                                            MaxCapacity = 1,
+                                            PalletNumber = 1,
+                                            TriggerLevel = 1
+                                        };
+
+            this.AuthorisationService.HasPermissionFor("stock-trigger-level.create", Arg.Any<IEnumerable<string>>()).Returns(true);
```

Comment:
I'd use your AuthorisedAction constants in here as opposed to strings, just to prevent future typos